style.css

```diff
@@ -7,6 +7,8 @@
 	--white-color:#fff;
 	--black-color:#000 ;
 	--favorite-color:rgba(255, 0, 0, 0.952);
+	--border-color:rgba(88, 85, 85, 0.384);
+	--grey-background-color:rgba(224, 224, 224, 0.31);
```

Comment:
suggestion: stick to hex or rgba. As Vikram says, consistency is key

Reply:
As I am using alpha so thought of keeping it in rgba format only, converted to hex code for consistency.